ServerMonitoringSystem/Services/IStatisticsCollector.cs

```diff
@@ -0,0 +1,6 @@
+﻿using ServerMonitoringSystem.Domain;
+
+public interface IStatisticsCollector
```

Comment:
Lets implement IDisposable to dispose all the resources allocation in the collector.